misc-utils/lsfd.c

```diff
@@ -499,6 +499,9 @@ static const struct file_class *stat2class(struct stat *sb)
 		if (major(dev) != 0)
 			return &file_class;
 
+		if (is_nsfs_dev(dev))
+			return &nsfs_file_class;
+
```

Comment:
Thank you for updating the pull request.

As far as seeing the result of the test, this change works well.
Now, we don't need the original condition for detecting nsfs anymonre:
```
fs = get_nodev_filesystem(minor(dev));
if (fs && strcmp(fs, "nsfs") == 0)
          return &nsfs_file_class;
```